<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserSettings</class>
 <widget class="QWidget" name="LaserSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QFormLayout">
   <item row="0" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Trail:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QSlider" name="persistence">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="widgets::ColorButton" name="color0">
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="locked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::ColorButton" name="color1">
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="color">
        <color>
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="locked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::ColorButton" name="color2">
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="color">
        <color>
         <red>0</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="locked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="widgets::ColorButton" name="color3">
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="color">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </property>
       <property name="locked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="trackpointer">
     <property name="text">
      <string>Track hovering pointer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgets::ColorButton</class>
   <extends>QToolButton</extends>
   <header>desktop/widgets/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>persistence</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>persistence</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
